CreativePower-alve/budget-app
# feat Edit budget

# Edit Budget

## What is this feature ? 

- Rename tabs and add new icons for them
- Use Context API to share global state across components
- Display initial budget properties for current user: currency, description, contribution
- Display total value from contributions of all users that share the budget 
- Add new charts library: [victory-native](https://formidable.com/open-source/victory/docs/native) charts
- Add [native-base](https://docs.nativebase.io/) UI framework library
 
## How to test ? 

- build app locally with yarn start
- Check that there are now two tabs : Expenses and Budget
- Check that on the Budget tab you can see existing property values for your budget: currency, description, contribution and that the total budget appears as well 
- Check that you can edit and save any of your budget properties
